config/locales/en.yml

```diff
@@ -18,6 +18,7 @@ en:
     important: Important
     success: Success
     warning: Warning
+    action_required: Action required
```

Comment:
For reviewers: I've added this new header under banners rather then reusing the existing [jobs.action_required.heading](https://github.com/DFE-Digital/teaching-vacancies/blob/master/config/locales/notifications.yml#L8) one, my logic being that it's a banner title. Is this the best thing to do?

Furthermore, should I delete `jobs.action_required.heading`, as it isn't being used anywhere now?

Reply:
Sounds good! Yeah, feel free to kill it.

Reply:
Yes you can remove it now that it's not used anymore